test/wallaby/experimental/chrome/logger_test.exs

```diff
@@ -0,0 +1,63 @@
+defmodule Wallaby.Experimental.Chrome.LoggerTest do
+  use ExUnit.Case, async: false
+
+  alias Wallaby.Experimental.Chrome.Logger
+  import ExUnit.CaptureIO
+
+  describe "parse_log/1" do
+    test "removes line numbers from the end of INFO logs" do
+      fun = fn ->
+        build_log(~s(http://localhost:52615//logs.html 13:14 "test"))
+        |> Logger.parse_log
+      end
+
+      assert capture_io(fun) == "test\n"
+    end
+
+    test "prints non-string data types" do
+      fun = fn ->
+        build_log(~s(http://localhost:52615//logs.html 13:14 1))
+        |> Logger.parse_log
+      end
+
+      assert capture_io(fun) == "1\n"
+
+      fun = fn ->
+        build_log(~s[http://localhost:52615//logs.html 13:14 Array(2)])
+        |> Logger.parse_log
+      end
+
+      assert capture_io(fun) == "Array(2)\n"
+    end
+
+    test "pretty prints json" do
+      message = ~s(http://localhost:54579//logs.html 13:14 "{\"href\":\"http://localhost:54579//logs.html\",\"ancestorOrigins\":{}}")
+
+      fun = fn ->
+        message
+        |> build_log
+        |> Logger.parse_log
+      end
+
+      assert capture_io(fun) == "\n{\n  \"href\": \"http://localhost:54579//logs.html\",\n  \"ancestorOrigins\": {}\n}\n"
+    end
+
+    test "can be disabled" do
+      Application.put_env(:wallaby, :js_logger, nil)
+
+      fun = fn ->
+        "test log"
+        |> build_log()
+        |> Logger.parse_log
+      end
+
+      assert capture_io(fun) == ""
+
+      Application.put_env(:wallaby, :js_logger, :stdio)
```

Comment:
#301 Also adds `Wallaby.SettingsTestHelper` which has a `ensure_setting_is_reset/2` that ensures the a setting is reset even if the test fails. Once that PR is merged, would you mind adapting the test to use this helper?

Reply:
Will do.